shawerma/views.py

```diff
@@ -0,0 +1,89 @@
+from django.db.models import Max,Count,Avg,Sum,F,Value
+from django.db.models.functions import TruncMonth
+from django.contrib.admin.views.decorators import staff_member_required
+from django.http import HttpResponse
+from django.contrib.auth import authenticate, login
+
+from rest_framework.permissions import IsAuthenticated,IsAdminUser
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import generics
+from rest_framework.response import Response
+
+from .serializers import MonthlyReport,AVGCustomer,ItemSerializers,MenuSerializers,OrderSerializers,HistorySerializers,BestCustomer
+from .models import User,MenuItem,Order
+
+
+class CreateView(generics.ListCreateAPIView):
+    queryset = MenuItem.objects.all()
+    serializer_class = MenuSerializers
+
+    def perform_create(self, serializer):
+        """Save the post data when creating a new bucketlist."""
+        serializer.save()
+
+class CreateNewOrder(generics.CreateAPIView):
+    serializer_class = OrderSerializers
+    queryset = Order.objects.all()
+
+    def perform_create(self, serializer):
+        """Save the post data when creating a new bucketlist."""
+        serializer.save(user=self.request.user)
+
+class CreateNewItem(generics.CreateAPIView):
+    serializer_class =ItemSerializers
+
+    def create(self, request, *args, **kwargs):
+        request.data
+
+
+class CreateHistory(generics.ListCreateAPIView):
+    queryset = ""
```

Comment:
how this can be empty string?! its a Queryset object... please remove it if not needed